src/Donations/index.tsx

```diff
@@ -19,6 +19,21 @@ import {
 } from "src/Utils/donationStepConstants";
 import { useRouter } from "next/router";
 import BackButton from "public/assets/icons/BackButton";
+import VerifiedIcon from "@mui/icons-material/Verified";
+import Popover from "@material-ui/core/Popover";
+import Typography from "@material-ui/core/Typography";
+import { makeStyles, createStyles, Theme } from "@material-ui/core/styles";
```

Comment:
"@material-ui/core" has been replaced with "@mui/material" during the next 13 upgrade, as it is not compatible with react 18.  Please use the current MUI dependencies. You will need to update the code, and also check that functionality/UX is as expected.